package.json

```diff
@@ -39,6 +42,9 @@
     "webpack-dev-server": "^1.16.1"
   },
   "dependencies": {
-    "jquery": "^3.1.1"
+    "jquery": "^3.1.1",
```

Comment:
just tried it. removed jquery dependency and it complained saying that "bootstrap requires jquery". I guess it was required after all.